jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -730,9 +731,31 @@ static Instant getCreationTime(String configuredCreationTime, ProjectProperties
           return Instant.now();
 
         default:
-          return DateTimeFormatter.ISO_DATE_TIME.parse(configuredCreationTime, Instant::from);
+          DateTimeFormatter formatter =
+              new DateTimeFormatterBuilder()
+                  .parseCaseInsensitive()
+                  .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
+                  // iso 8601 strict
+                  .optionalStart()
+                  .appendOffset("+HH:MM", "+00:00")
+                  .optionalStart()
+                  .appendLiteral('[')
+                  .parseCaseSensitive()
+                  .appendZoneRegionId()
+                  .appendLiteral(']')
+                  .optionalEnd()
```

Comment:
Judging from this [Javadoc](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME), shouldn't this be after `.appendOffset("+HH", "Z")`?

Reply:
I was trying to recreate ISO_DATE_TIME here, but this whole section seems unnecessary.